src/main/java/org/folio/rest/impl/ScheduledNoticesAPI.java

```diff
@@ -0,0 +1,162 @@
+  public void postScheduledNoticeStorageScheduledNotices(String lang,
+                                                         ScheduledNotice entity,
+                                                         Map<String, String> okapiHeaders,
+                                                         Handler<AsyncResult<Response>> asyncResultHandler,
+                                                         Context vertxContext) {
+
+    PgUtil.post(SCHEDULED_NOTICE_TABLE, entity, okapiHeaders, vertxContext,
+      PostScheduledNoticeStorageScheduledNoticesResponse.class, asyncResultHandler);
+  }
+
+  @Override
+  public void getScheduledNoticeStorageScheduledNoticesByScheduledNoticeId(String scheduledNoticeId,
+                                                                           String lang,
+                                                                           Map<String, String> okapiHeaders,
+                                                                           Handler<AsyncResult<Response>> asyncResultHandler,
+                                                                           Context vertxContext) {
+
+    PgUtil.getById(SCHEDULED_NOTICE_TABLE, ScheduledNotice.class, scheduledNoticeId, okapiHeaders, vertxContext,
+      GetScheduledNoticeStorageScheduledNoticesByScheduledNoticeIdResponse.class, asyncResultHandler);
+  }
+
+  @Override
+  public void deleteScheduledNoticeStorageScheduledNoticesByScheduledNoticeId(String scheduledNoticeId,
+                                                                              String lang, Map<String, String> okapiHeaders,
+                                                                              Handler<AsyncResult<Response>> asyncResultHandler,
+                                                                              Context vertxContext) {
+
+    PgUtil.deleteById(SCHEDULED_NOTICE_TABLE, scheduledNoticeId, okapiHeaders, vertxContext,
+      DeleteScheduledNoticeStorageScheduledNoticesByScheduledNoticeIdResponse.class, asyncResultHandler);
+
+  }
+
+  @Override
+  public void putScheduledNoticeStorageScheduledNoticesByScheduledNoticeId(String scheduledNoticeId,
+                                                                           String lang,
+                                                                           ScheduledNotice entity,
+                                                                           Map<String, String> okapiHeaders,
+                                                                           Handler<AsyncResult<Response>> asyncResultHandler,
+                                                                           Context vertxContext) {
+
+    PgUtil.put(SCHEDULED_NOTICE_TABLE, entity, scheduledNoticeId, okapiHeaders, vertxContext,
+      PutScheduledNoticeStorageScheduledNoticesByScheduledNoticeIdResponse.class, asyncResultHandler);
+
+  }
+
+  private Future<Void> executeSql(PostgresClient pgClient, String sql) {
+
```

Comment:
Why is there a blank line here?

Reply:
For me it seems easier to read, when method's signature and body are split by a blank line.

Reply:
I see different examples in mod-circulation-storage. Some methods have a blank line, but some don't.

It would be good to keep coding style consistency for the module. Probably it is better to remove a blank line. What do you think?

Reply:
I personally think that decision is highly contextual.

If a method has lots of parameters, I find the whitespace often helps to distinguish the definition from the implementation.

If a method is small and simple, I find the whitespace distracting.

I think agreeing on a consistent set of guidance would be useful. That is likely a broader topic of discussion (and likely relates to method parameter indentation, which is also important and inconsistent).